### PR TITLE
Implementing game window

### PR DESCRIPTION
**Pull request by emersion**

### CODE REVIEW

**Comment by emersion** [comment]
TODO: remove `typedef std::string string;`, that makes the build fail.

**Comment by redianthus** [comment]
Looks like some tests don't pass... :/

File: gui/music_line.hpp [hunk 1]
```diff
@@ -4,7 +4,9 @@
 #include <iostream>
 #include <QWidget>
 #include "game_window.hpp"
-typedef std::string string;
```
Comment (redianthus) [inline gui/music_line.hpp]: @emersion, are you the one who wrote this ?

Comment (emersion) [reply gui/music_line.hpp]: [`git blame`](https://github.com/fanfaresimulator/fanfaresimulator/blame/177c3c99fe1c46c5d37d15d30809265e22f12288/gui/music_line.hpp) 🤔

Comment (redianthus) [reply gui/music_line.hpp]: I don't want to blame the author. That's amazing.

Comment (emersion) [reply gui/music_line.hpp]: `git praise`?

Comment (redianthus) [reply gui/music_line.hpp]: capture du 2017-12-06 14-43-26

:'(

Comment (emersion) [reply gui/music_line.hpp]:
```shell
git config --global alias.praise blame
```

👍

**Pull request merged**